GDJS/Runtime/variablescontainer.js

```diff
@@ -10,7 +10,7 @@
  *
  * @memberof gdjs
  * @class VariablesContainer
- * @param {Object} initialVariablesData Optional object containing initial variables.
+ * @param {Array<VariableData>} [initialVariablesData] Optional Array containing representations of the base variables.
```

Comment:
No capital to Array (Optional array containg [...])

Reply:
fixed